index.js

```diff
@@ -59,13 +58,13 @@ lib.preFlow(async function(err, results) {
     return cmd._name;
   });
 
-  if (!program.args.length) {
-    console.log('resume-cli:'.cyan, 'https://jsonresume.org', '\n');
+  if (program.rawArgs.length < 3) {
```

Comment:
What is the significance of `3` here? Would it make sense to explain with a code comment?

Reply:
It is how to test for no arguments in Commander 4.0.0+, see [migration tips](https://github.com/tj/commander.js/blob/master/CHANGELOG.md#migration-tips-1).
I can add a comment, but I thought it was unnecessary as it is a documented pattern.

Reply:
A link to the docs would be sufficient.

Reply:
Done 👍